src/cross-origin/requests.conf

```diff
@@ -9,5 +9,8 @@
 # http://www.w3.org/TR/cors/
 
 <IfModule mod_headers.c>
+    # Only for you domain
+    # SetEnvIf Host ^(.*)$ YOU_HOST=$1
```

Comment:
What's the reason for the SetEnvIf call? 

There's also a typo on line 14 `http://%{YOU_HOST}e` <-